client/index.test.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import toJson from 'enzyme-to-json'
+import { shallow } from 'enzyme'
+import Day from './components/Day.jsx'
+
+
+describe('Day', () => {
+  it('displays text based on props.day', () => {
+    const day = shallow(<Day day={1}/>);
+    
+    expect(day.text()).toBe('1');
+  });
+});
```

Comment:
Awesome! Congrats on your first test! Code is clean and easy to read. Only comment I have is to always end your code with a new line at the end of it. Once you set up your eslint style guides for hack reactor and airbnb these little syntax and styling errors won't be a problem anymore.